lib/src/chart/chart.dart

```diff
@@ -213,6 +214,19 @@ class _ChartState<D> extends State<Chart<D>> {
   /// Details of previous scale update.
   ScaleUpdateDetails? gestureScaleDetail;
 
+  /// Keep track if there is a mouse connected.
+  late bool _mouseIsConnected;
+
+  @override
+  void initState() {
```

Comment:
I followed the approach of Flutter's tooltips. I check if theres a mouse, and if yes, I add a MouseRegion.